### ISSUE TITLE
norm[-1,1]for net='vgg'?

### ISSUE THREAD

**Issue by DISAPPEARED13** [richzhang/PerceptualSimilarity#84]
I found that you mention that we need to normalize the RGB images range into [-1, 1], but I saw that example worked at when ```net='alex'```, I wanna know that if this work for net='vgg', too?
Because I found some comments in https://discuss.pytorch.org/t/how-to-preprocess-input-for-pre-trained-networks/683/2 , @smth said that input should be norm to [0, 1], and now I am confused...
And I noticed that you also load your own pretrained network by giving "net" and "version", are you just use the framework defined by torchvision.model.vgg16 and then you trained it with ImageNet by norm to [-1, 1]?
sorry to bother and thanks a lot!  : )
